calculator.js

```diff
@@ -1,28 +1,69 @@
+// const printInputs = function(num1, num2)
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+const addCalc = function(num1, num2) {
+  result = (num1 * 1) + (num2 * 1)
+  console.log(`${num1} + ${num2} = ${result}`)
+};
+
+const subCalc = function(num1, num2) {
+  result =  (num1 * 1) - (num2 * 1)
+  console.log(`${num1} - ${num2} = ${result}`)
+};
+
+const multCalc = function(num1, num2) {
+  result = (num1 * 1) * (num2 * 1)
+  console.log(`${num1} * ${num2} = ${result}`)
+};
+
+const divCalc = function(num1, num2) {
+  if (!(num2 * 1)){
+    console.log("You cannot divide by zero")
+  } else 
+    result = (num1 * 1) / (num2 * 1)
+    console.log(`${num1} / ${num2} = ${result}`)
 }
```

Comment:
oops you are missing a set of curly braces!
```suggestion
  } else {
    result = (num1 * 1) / (num2 * 1)
    console.log(`${num1} / ${num2} = ${result}`)
  }
}
```